### ISSUE TITLE
Problem with curriculum learning?

### ISSUE THREAD

**Issue by PietroLodiRizzini** [chnsh/DCRNN_PyTorch#26]
the decoder of the `DCRNNModel` in `model/pytorch/dcrnn_model.py` seems to be fed with its own output when `use_curriculum_learning` is set to `False`:

```
for t in range(self.decoder_model.horizon):
            decoder_output, decoder_hidden_state = self.decoder_model(decoder_input,
                                                                      decoder_hidden_state)
            decoder_input = decoder_output
            outputs.append(decoder_output)
            if self.training and self.use_curriculum_learning:
                c = np.random.uniform(0, 1)
                if c < self._compute_sampling_threshold(batches_seen):
                    decoder_input = labels[t]
```

However, I think it should be fed with ground truth labels, isn't it?
cfr. DCRNN paper, page 4 1st line
